dayrepo/snippets/templates/snippet_post.html

```diff
@@ -0,0 +1,12 @@
+
+{% block page_title %}投稿{% endblock %}
+{% block title %}投稿{% endblock %}
+
+{% block content %}
+    <form method="post">
+        {{ form.as_p }}
+        {{ form_trouble.as_p }}
+        {% csrf_token %}
+        <button type="submit" class="button">投稿</button>
```

Comment:
class名の抽象度が適切ではないように感じます。
```suggestion
        <button type="submit" class="suippet_button">投稿</button>
```

Reply:
どうせならsnippet_buttonにしたいのでこちらで変えときます